### PR TITLE
1) added FRIENDSHIP_IS_MAGIC feature to remove PWNED from the data_ta…

### PR DESCRIPTION
**Pull request by lhayhurst**
…ble output and anonymize the PWNED in the recent output to be my little pony character names. 2) added small unit test to test the above 3) updated README with instructions on how to activate FRIENDSHIP_IS_MAGIC mode.

### CODE REVIEW

**Comment by heimdall-asguard** [comment]
Reviews      | 0/1 | Message
------------ | ------------- | ----------




[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/pwnbot&ref=c60ec099b1e1420ccbd1838dcf33619539dc22ae)

File: src/index.js [hunk 1]
```diff
@@ -123,6 +126,39 @@ exports.item_to_pwn = (item) => {
...
+  return array;
+}
+
+function usingAnonymousPwnees() {
+  return process.env.FRIENDSHIP_IS_MAGIC != null && process.env.FRIENDSHIP_IS_MAGIC == "1";
+}
+
+function getPwnedName(pwned, anonymousPwnees, index) {
+  if (anonymousPwnees ) {
+    if (index > anonymousPwnees.length-1) { //in case the pwn list ever grows out of the hardcoded 5 value and thre are more recent pwnees than the number of my little pony characters
```
Comment (krobertson) [inline src/index.js]: nit: `there` misspelled, and looks like the list now has 12 values. The comment does make for a long line that gets wrapped.

**Comment by heimdall-asguard** [comment]
Reviews      | 1/1 | Message
------------ | ------------- | ----------
krobertson | 1 | approved 

Pull request is reviewed and can be merged to master!

[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/pwnbot&ref=c60ec099b1e1420ccbd1838dcf33619539dc22ae)

File: src/index.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ var qs = require('qs')
 var moment = require('moment')
 var post = Promise.promisify(require('request').post)
 
+var myLittlePwnees = [ 'Twilight Sparkle', 'Rarity', 'Pinkie Pie', 'Applejack', 'Fluttershy', 'Rainbow Dash',
```
Comment (grahamjenson) [inline src/index.js]: I think these should be emojis.

File: src/index.js [hunk 3]
```diff
@@ -123,6 +126,39 @@ exports.item_to_pwn = (item) => {
...
+    temporaryValue = array[currentIndex];
+    array[currentIndex] = array[randomIndex];
+    array[randomIndex] = temporaryValue;
+  }
+
+  return array;
+}
+
+function usingAnonymousPwnees() {
+  return process.env.FRIENDSHIP_IS_MAGIC != null && process.env.FRIENDSHIP_IS_MAGIC == "1";
```
Comment (grahamjenson) [inline src/index.js]: Don't want the option, should just be default.

**Comment by itstehkman** [comment]
@grahamjenson should we merge this? ;)

**Pull request closed without merging**